lib/roby/actions/models/interface_base.rb

```diff
@@ -85,6 +85,20 @@ def describe(doc = nil)
             # interface. For instance, the start_all_devices action would create
             # a simple StartAllDevices task model.
             def register_action(name, action_model)
+                unless action_model.respond_to?(:to_action_model)
+                    raise ArgumentError, "register_action expects an action model, "\
+                        "got #{action_model.class} instead"
+                end
+
+                action_model = action_model.to_action_model
```

Comment:
```suggestion
                action_model = action_model.to_action_model.dup
```
Alguma razão pra separar em duas etapas?

Reply:
Na verdade, acho que não entendi porque você faria uma duplicada se quer registrar o modelo e não uma cópia.

Reply:
Please stick to English on the public repositories.

The action model is modified in-place in `register_action!`. Without the dup, we would modify all copies of the model (since this change is meant to add the ability to share actions across action interfaces).

Maybe add a comment about that @alcantara09 ?

Reply:
Yeah.. Everyone just found out @doudou can speak portuguese :P

Reply:
Me ? No. Not at all. Who would even think something like that.

Reply:
@doudou ops! I have not realized which repository was, my apologies.